wporg-gp-translation-events.php

```diff
@@ -65,7 +65,7 @@ function register_event_post_type() {
 		'menu_icon'   => 'dashicons-calendar',
 		'supports'    => array( 'title', 'editor', 'thumbnail', 'revisions' ),
 		'rewrite'     => array( 'slug' => $slug ),
-		'show_ui'     => false,
+		'show_ui'     => true,
```

Comment:
Since you needed to activate that, should we just use a filter here so that you can override it with another plugin?